src/app/pages/office/components/eco-card/eco-card.component.html

```diff
@@ -0,0 +1,14 @@
+<ion-card id="eco" *ngIf="(ecoFaults | async) && (ecoDisplay | async)">
+  <ion-card-content>
+    <ion-grid>
+      <ion-row class="ion-align-items-start mes-data">
+        <ion-col size="32">
+          <h4 class="fault-heading">ECO</h4>
+        </ion-col>
+        <ion-col class="padding-down ion-align-self-start ion-padding-start">
+          <div id="ecoFaults">{{ecoFaults | async}}</div>
```

Comment:
This should not need the async pipe, we should not be passing down observables into the component, we should be resolving them at the higher level